### PR TITLE
docs(project): add project contributor specific files and github templates

### PR DESCRIPTION
**Pull request by prateekpandey14**
**What this PR does**:
PR adds project specific files requires to maintain the project following cncf guidelines.

Added files are :
- github issue template
- github feature--enhancement-request template
- github pull request template
- CONTRIBUTING.md
- RELEASE.md
- CODE_OF_CONDUCT.md
- GOVERNANCE.md
- MAINTAINERS
- code-standard.md
- developer-setup.md

Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

### CODE REVIEW

File: docs/developer-setup.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+# Confirm that your remotes make sense:
+git remote -v
+```
+> **Note:** If your `GOPATH` has more than one (`:` separated) paths in it, then you should use *one of your go path* instead of `$GOPATH` in the commands mentioned here. This statement holds throughout this document.
+
+### Building and Testing your changes
+
+* To build the cstor csi-driver binary
+```sh
```
Comment (utkarshmani1997) [inline docs/developer-setup.md:45]: nit: is sh required?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```
Comment (kmova) [inline CODE_OF_CONDUCT.md:1]: Use this format please. https://github.com/openebs/linux-utils/blob/master/CODE_OF_CONDUCT.md

File: docs/developer-setup.md (hunk 1 above)
Comment (prateekpandey14) [reply docs/developer-setup.md:45]: its just choice how you want to show it, its fine  guess

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (prateekpandey14) [reply CODE_OF_CONDUCT.md:1]: I just followed the other openebs repo having a similar format, should i change to use as you mentioned above ?

Comment (kmova) [reply CODE_OF_CONDUCT.md:1]: yes.

Comment (kmova) [reply CODE_OF_CONDUCT.md:1]: we will have to change others as well

Comment (prateekpandey14) [reply CODE_OF_CONDUCT.md]: done

**Pull request merged**